src/js/components/collapse.js

```diff
@@ -64,6 +92,11 @@ class Collapse {
     emitter.emit('collapse:hide', this.element, this.toggle)
   }
 
+  /**
+   * Set `isCollapsed` flag to false
```

Comment:
Can you use something to connect the sentences? You can separate them with commas or finish each sentence with a period `.` and then start the others with capital letters. Think about generating a documentation page for this component using a tool like JSDoc for instance. If you write sentences in this way you're going to end up with a grammatically incorrect paragraph for that documentation.